### PR TITLE
Add a stress test for reading

### PR DESCRIPTION
**Pull request by sticnarf**
### What is changed and how does it work?

This PR adds a stress test for the new unified thread pool in TiKV.

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

Code changes

 - Has Go code change

Side effects

Related changes

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

**Comment by mahjonp** [comment]
@sticnarf CI was failed. PTAL

File: cmd/read-stress/main.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+import (
+	"context"
+	"flag"
+	"github.com/pingcap/tipocket/cmd/util"
+	"github.com/pingcap/tipocket/pkg/cluster"
+	"github.com/pingcap/tipocket/pkg/control"
+	"github.com/pingcap/tipocket/pkg/test-infra/fixture"
+	"github.com/pingcap/tipocket/pkg/test-infra/tidb"
+	stress "github.com/pingcap/tipocket/tests/read-stress"
```
Comment (mahjonp) [inline cmd/read-stress/main.go:24]: please group imports here.

File: tests/read-stress/stress.go [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+	"context"
+	"database/sql"
+	"fmt"
+	"github.com/ngaut/log"
+	"github.com/pingcap/tipocket/util"
+	"math/rand"
+	"sync"
+	"time"
+
+	"github.com/pingcap/tipocket/pkg/cluster/types"
```
Comment (mahjonp) [inline tests/read-stress/stress.go:13]: ditto

File: cmd/read-stress/main.go (hunk 1 above)
Comment (sticnarf) [reply cmd/read-stress/main.go]: I think they're now grouped like other files.

**Comment by sticnarf** [comment]
@mahjonp Could you please help me see if there's anything wrong with the configurations? I'm new to tipocket.

**Comment by mahjonp** [comment]
> @mahjonp Could you please help me see if there's anything wrong with the configurations? I'm new to tipocket.

Sorry for your inconvenience, I find nothing is wrong.

**Comment by sticnarf** [comment]
Looks like the test has passed but the status is not reported successfully.

**Pull request merged**